api/go.mod

```diff
@@ -8,11 +8,13 @@ require (
 	github.com/jinzhu/gorm v1.9.12
 	github.com/joho/godotenv v1.3.0
 	github.com/lib/pq v1.3.0
+	github.com/prometheus/common v0.10.0
```

Comment:
are we using `prometheus`? you can run `go mod tidy`. It will remove unused dependencies